### ISSUE TITLE
Accessibility: Bar chart keyboard navigation inverted

### ISSUE THREAD

**Issue by philipstarkey** [highcharts/highcharts#23687]
Bar charts reverse their x-axis direction by default. If, however, you change that default and you set `xAxis: { reversed: false }`, then keyboard navigation using up/down arrows is visually inverted.

#### Expected behaviour
The up arrow should select the bar visually above the currently focussed bar.
The down arrow should select the bar visually below the currently focussed bar.
This behaviour should be independent of whether the x-axis is reversed. Keyboard navigation direction should always be based on visual order.

#### Actual behaviour
When the x-axis is not reversed on a bar chart, the keyboard navigation is visually inverted.

#### Live demo with steps to reproduce
https://jsfiddle.net/03hv1qds/1/

#### Product version
Highcharts core 12.4

#### Affected browser(s)
At least Chrome and Safari. I have no reason to think it's browser specific.

**Comment by johngwilkinson** [comment]
Hopefully it's appropriate for me to join in here: this may be the same bug I was looking to report, which is that keyboard navigation does not match visual order when a column chart has a reversed x-axis.

Demo: https://jsfiddle.net/xruL0kst/1/

In the demo, you can see that the arrow keys move focus in the wrong direction between bars: the left arrow moves right, and the right arrow moves left.

As OP says, "Keyboard navigation direction should always be based on visual order."

**Comment by magdalena-gut** [comment]
Thanks for reporting this!

The current [normal mode](https://api.highcharts.com/highcharts/accessibility.keyboardNavigation.seriesNavigation.mode) for keyboard navigation doesn’t always follow the visual layout, especially when the axes are reversed. We plan to improve this behavior as part of the upcoming accessibility module updates.

As a temporary workaround, you can try setting the mode to `serialize`. While it won’t fully match the expected visual navigation, it should behave more consistently in this case.
